.markdownlint.json

```
@@ -0,0 +1,4 @@
+{
+  "default": true,
+  "MD013": false
```

```suggestion
  "line-length": false
```

Use the human-readable name for maintainability

---

Fixed. And thank you. That's a lot easier to read.